test/core/bitswap.spec.js

```diff
@@ -16,10 +16,9 @@ const isNode = require('detect-node')
 const multihashing = require('multihashing-async')
 const CID = require('cids')
 
-const DaemonFactory = require('ipfsd-ctl')
-const df = DaemonFactory.create({ type: 'js' })
-
-const dfProc = DaemonFactory.create({ type: 'proc' })
+const IPFSFactory = require('ipfsd-ctl')
+const fDaemon = IPFSFactory.create({ type: 'js' })
+const fInProc = IPFSFactory.create({ type: 'proc' })
```

Comment:
I find `fInProc` a strange, hard to read, meaningless, uncommon name. What about just `InProc` or `InProcDeamon`.

Reply:
Same as https://github.com/ipfs/js-ipfs/pull/1215#discussion_r169066092. I can use the full word if that works better :)